cmd/treedb-native-server/main.go

```
@@ -0,0 +1,93 @@
+		log.Fatalf("Failed to open TreeDB leaf-page log: %v", err)
+	}
+	defer leafLog.Close()
+	database.SetLeafPageLog(leafLog)
+
+	manager := collections.NewCollectionManager(database)
+	server := nativewire.NewServer(nativewire.ServerOptions{
+		Collections: manager,
+		Backend:     database,
+	})
+
+	ln, err := net.Listen("tcp", *addr)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)